index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>Portfolio-setup-and-mobile-first</title>
+</head>
+<body>
+    <!-- the header part -->
+    <section class="head">
+        <header>
+            <div class="logo">
+                <a href="#">My Logo</a>
+            </div>
+            <div class="navbar">
+                <i class="fa fa-bars"></i>
+            </div>
+        </header>
+    </section>
+
+    <!-- upper - background content -->
+
+    <!-- the text part -->
+
+    <section class="text">
+        <h1>I’m Ibrahim
+            Glad to see you!
+        </h1>
+        <p>I’m a software developer! I can help you 
+            build a product , feature or website Look
+             through some of my work and experience! 
+             If you like what you see and have a project 
+             you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-text">
+            <h5><b>LET'S CONNECT</b></h5>
+            
+        </div>
+        <div class="social-media">
+          <a href="https://twitter.com/kingibro345"><img src="twitter.svg" alt="" target=_blank /></a>  
+          <a href="https://linkedin.com/ibrahima"><img src="Linkedin icon.svg" alt="" target=_blank /></a>  
+          <a href="https://medium.com/"><img src="medium.svg" alt="" target=_blank /></a>  
+          <a href="https://github.com/kingibro345"><img src="git.svg" alt="" target=_blank /></a>  
+          <a href="#"><img src="twinny.svg" alt=""/></a>  
+        </div>
```

Comment:
- [OPTIONAL] You don't need to remove the `ul` that used to group your social media icons, but what the previous reviewer was trying to say is that you should make each of the icons a link to your social media page. Look at the example below and kindly apply it to this section of your project.
```
<ul>
  <li>
        <a href="https://twitter.com/kingibro345">
            <img src="twitter.svg" alt="" target=_blank />
        </a>
  </li>
  <li>
      the rest icons.......
  </li>
</ul>
```